jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -473,7 +478,11 @@ static void completeInitialization() {
 	/*[ENDIF]*/ // Sidecar18-SE-OpenJ9
 
 	/*[IF (Sidecar18-SE-OpenJ9|Sidecar19-SE)&!(PLATFORM-mz31|PLATFORM-mz64)]*/
-	setIn(new BufferedInputStream(new FileInputStream(FileDescriptor.in)));
+	InputStream tempIn = new BufferedInputStream(new FileInputStream(FileDescriptor.in));
+	setIn(tempIn);
+/*[IF JAVA_SPEC_VERSION >= 20]*/
+	initialIn = tempIn;
+/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
```

Comment:
Please indent the preprocessor comments to match nearby instances (e.g. lines 488, 494-498).

Reply:
Updated in my commit.
FYI: @JasonFengJ9